### PR TITLE
Longview

### PR DESCRIPTION
**Pull request by yoshizzle**

### CODE REVIEW

File: docs/src/data/objects/linode_status.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+enums:
+  Status:
+    offline: The Linode is powered off.
+    booting: The Linode is currently booting up.
+    running: The Linode is currently running.
+    shutting_down: The Linode is currently shutting down.
+    rebooting: The Linode is rebooting.
+    provisioning: The Linode is being created.
+    deleting: The Linode is being deleted.
+    migrating: The Linode is being migrated to a new host/region.
```
Comment (Dorthu) [inline docs/src/data/objects/linode_status.yaml]: 👎

Comment (yoshizzle) [reply docs/src/data/objects/linode_status.yaml]: very weird. Not sure how that creeped in there. I'll remove.

File: docs/src/data/objects/longview.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# vim: set et ts=2 sw=2 tw=80 :
+name: Longview
```
Comment (Dorthu) [inline docs/src/data/objects/longview.yaml]: I would prefer you rename this file to `longviewclient.yaml` and call this a LongviewClient.

File: docs/src/data/endpoints/longview.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            curl -H "Authorization: Bearer $TOKEN" \
+                https://$api_root/$version/longview/clients
+  /longview/clients/$id:
+    type: resource
+    authenticated: true
+    description: >
+      Manage a particular Longview client your account may access.
+    methods:
+      GET:
+        response: longview
```
Comment (Dorthu) [inline docs/src/data/endpoints/longview.yaml]: I would call this a longviewclient

File: docs/src/data/objects/longviewclients.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# vim: set et ts=2 sw=2 tw=80 :
+name: Longview
```
Comment (Dorthu) [inline docs/src/data/objects/longviewclients.yaml]: Please rename this object to LongviewClient

File: docs/src/data/endpoints/longviewclients.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+            curl -H "Authorization: Bearer $TOKEN" \
+                https://$api_root/$version/longview/clients
+  /longview/clients/$id:
+    type: resource
+    authenticated: true
+    description: >
+      Manage a particular Longview client your account may access.
+    methods:
+      GET:
+        response: longview
```
Comment (Dorthu) [inline docs/src/data/endpoints/longviewclients.yaml]: please rename this to longviewclient

File: docs/src/data/endpoints/longviewclients.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+description: >
+  Longview endpoints provide a means of managing Longview client objects on your account.
+endpoints:
+  /longview/clients:
+    authenticated: true
+    description: >
+      Manage the collection of Longview client objects your account may access.
+    methods:
+      GET:
+        response: longview
```
Comment (Dorthu) [inline docs/src/data/endpoints/longviewclients.yaml]: please rename this to longviewclient

**Pull request merged**